test/common/network/connection_impl_test.cc

```diff
@@ -1563,7 +1567,10 @@ TEST_P(ReadBufferLimitTest, SomeLimit) {
 
 class TcpClientConnectionImplTest : public testing::TestWithParam<Address::IpVersion> {
 protected:
-  TcpClientConnectionImplTest() : dispatcher_(time_system_) {}
+  TcpClientConnectionImplTest()
+      : api_(Api::createApiForTest(stats_store_)), dispatcher_(time_system_, *api_) {}
+  Stats::IsolatedStoreImpl stats_store_;
```

Comment:
Nit: prefer a blank line between constructor/destructors and the member variables (here and elsewhere).